git_machete/tests/functional/test_machete.py

```diff
@@ -43,7 +47,7 @@ def __init__(self) -> None:
 class MockGithubAPIState:
     def __init__(self, pulls: List[Dict[str, Any]]) -> None:
         self.pulls: List[Dict[str, Any]] = pulls
-        self.user: Dict[str, str] = {'login': 'github_user', 'type': 'User', 'company': 'VirtusLab'}
+        self.user: Dict[str, str] = {'login': 'other_user', 'type': 'User', 'company': 'VirtusLab'}  # login must be different to one used in pull requests, otherwise pull request author will not be annotated
```

Comment:
Nit:
```suggestion
        self.user: Dict[str, str] = {'login': 'other_user', 'type': 'User', 'company': 'VirtusLab'}  # login must be different from the one used in pull requests, otherwise pull request author will not be annotated
```